src/components/AppIcon/assets/app-default.svg

```diff
@@ -1,20 +1 @@
-<svg width="80" height="80" viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M80 0H0V80H80V0Z" fill="url(#paint0_linear)"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M40.5184 15.3425L18.6301 27.9452V53.1507L40.5184 65.7534L62.4077 53.1507V27.9452L40.5184 15.3425Z" fill="url(#paint1_linear)"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M40.5184 65.7534V15.3425L62.4077 27.9452V53.1507L40.5184 65.7534Z" fill="#73F0F8"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M18.6301 27.9452L40.5184 15.3425L62.4077 27.9452L40.5184 39.9814L18.6301 27.9452Z" fill="url(#paint2_linear)"/>
-<defs>
-<linearGradient id="paint0_linear" x1="0" y1="0" x2="80" y2="75.8648" gradientUnits="userSpaceOnUse">
-<stop stop-color="#33BCE6"/>
-<stop offset="1" stop-color="#48E2E5"/>
-</linearGradient>
-<linearGradient id="paint1_linear" x1="20.8829" y1="27.5449" x2="44.8904" y2="62.7744" gradientUnits="userSpaceOnUse">
-<stop stop-color="#2597B7"/>
-<stop offset="1" stop-color="#39C5E1"/>
-</linearGradient>
-<linearGradient id="paint2_linear" x1="21.1421" y1="27.6619" x2="62.4077" y2="27.6619" gradientUnits="userSpaceOnUse">
-<stop stop-color="#3DCEE5"/>
-<stop offset="1" stop-color="#48E2E6"/>
-</linearGradient>
-</defs>
-</svg>
+<svg width="80" height="80" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M80 0H0v80h80V0z" fill="url(#paint0_linear)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M40.518 15.342L18.63 27.945v25.206l21.888 12.602 21.89-12.602V27.945l-21.89-12.603z" fill="url(#paint1_linear)"/><path fill-rule="evenodd" clip-rule="evenodd" d="M40.518 65.753v-50.41l21.89 12.602v25.206l-21.89 12.602z" fill="#73F0F8"/><path fill-rule="evenodd" clip-rule="evenodd" d="M18.63 27.945l21.888-12.603 21.89 12.603-21.89 12.036L18.63 27.945z" fill="url(#paint2_linear)"/><defs><linearGradient id="paint0_linear" x1="0" y1="0" x2="80" y2="75.865" gradientUnits="userSpaceOnUse"><stop stop-color="#33BCE6"/><stop offset="1" stop-color="#48E2E5"/></linearGradient><linearGradient id="paint1_linear" x1="20.883" y1="27.545" x2="44.89" y2="62.774" gradientUnits="userSpaceOnUse"><stop stop-color="#2597B7"/><stop offset="1" stop-color="#39C5E1"/></linearGradient><linearGradient id="paint2_linear" x1="21.142" y1="27.662" x2="62.408" y2="27.662" gradientUnits="userSpaceOnUse"><stop stop-color="#3DCEE5"/><stop offset="1" stop-color="#48E2E6"/></linearGradient></defs></svg>
```

Comment:
Should this also be 56x56?

Reply:
Done, I guess it’s time to update the guidelines :D